src/ExpressionEvaluator/CSharp/Source/ResultProvider/CSharpFormatter.TypeNames.cs

```diff
@@ -51,6 +52,32 @@ protected override void AppendIdentifierEscapingPotentialKeywords(StringBuilder
             builder.Append('>');
         }
 
+        protected override void AppendTupleFields(
```

Comment:
We're not including default names (`Item1`, ...) in Value or Type, but we will include friendly names if any. I'll handle that in a separate PR.